### PR TITLE
Dedup and strip long commands for templated and templated filters GT …

### PR DESCRIPTION
**Pull request by Rebecca-Qian**
# Description

A simple data processing script that filters large data files to create ground truth annotations.
- Deduplicate commands
- Remove long commands (>6 words)
- Remove NOOPs

Overview of templated data:
- 56% are unique commands
- 40% are short commands

Selected 10% of templated and 54% of templated filters for use in ground truth.

Fixes # (issue)
#20 

## Type of change
Please check the options that are relevant.

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [x] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

If this PR introduces a change or fixes a bug, please add before and after screenshots (if this is causes a UX change) or before and after examples(this could be plain text, videos etc ) to demonstrate the change.

If this PR introduces a new feature, please also add examples or screenshots demonstrating the feature.

# Testing

Please describe the tests that you ran to verify your changes and how we can reproduce.

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [x] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added relevant collaborators to review the PR before merge.

### CODE REVIEW

File: tools/data_processing_scripts/process_templated_for_gt.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+"""
+This script scrapes player logs from S3 and grabs the commands that are most played.
```
Comment (kavyasrinet) [inline tools/data_processing_scripts/process_templated_for_gt.py:2]: A nit: This is not what the script does :)
Let's rewrite the docstring ?

**Review by snyxan — APPROVED** [review]
LGTM bc it speeds up agent initialization.
related: #20

File: tools/data_processing_scripts/process_templated_for_gt.py (hunk 1 above)
Comment (Rebecca-Qian) [reply tools/data_processing_scripts/process_templated_for_gt.py:2]: Good catch! 

BTW, I'm going to be checking in more data processing scripts in `tools/data_processing_scripts` for reuse and to automate more aspects of data processing/model training.

Comment (Rebecca-Qian) [reply tools/data_processing_scripts/process_templated_for_gt.py:2]: I had started scraping the logs..but quickly saw the bigger problem was duplicates. Keeping this in mind for future improvements :)

**Review by kavyasrinet — APPROVED** [review]

**Pull request merged**